Documentation/cli-prerequisites.md

```diff
@@ -0,0 +1,81 @@
+CLI native prerequisites
+=========================
+
+This document outlines the dependencies needed to run .NET Core CLI tools. Most of these dependencies are also .NET Core's general dependencies, so installing them will make sure that you can run applications written for .NET Core in general.
+
+## Windows dependencies
+On Windows, the only dependency is the VC++ Redistributable. Depending on the version of Windows you are running on, the versions are changing.
+
+* Windows 10
+    * [Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)
+* Windows 7 & 8, Windows Server 2008 & 2012
+    *  [Visual C++ Redistributable for Visual Studio 2012 Update 4](https://www.microsoft.com/en-us/download/confirmation.aspx?id=30679)
+
+## Ubuntu
+Ubuntu distributions require the following libraries installed:
+
+* libc6
+* libedit2
+* libffi6
+* libgcc1
+* libicu52
+* liblldb-3.6
+* libllvm3.6
+* liblttng-ust0
+* liblzma5
+* libncurses5
+* libpython2.7
```

Comment:
Possibly.  FWIW, It feels really weird that we have this giant matrix of dependencies for Ubuntu and very few for CentOS.  Especially given that things don't line up (e.g. unzip is not listed here, we require random RPM tools there?).  We also seem to require -revel dependencies on CentOS which also feels wrong.

Note that LLDB is not required for normal CLI operation (you won't be able to debug the runtime using the SOS plugin but that's not something most folks are going to want to do).